pkgs/applications/virtualization/toolbox/default.nix

```diff
@@ -0,0 +1,62 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, go-md2man
+, podman
+}:
+buildGoModule rec {
+  pname = "toolbox";
+  version = "0.0.99";
+
+  src = fetchFromGitHub {
+    owner = "containers";
+    repo = "toolbox";
+    rev = version;
+    sha256 = "CKjndWXDLxQ5epco/+TwRFxftbeMhYp/7NOWglF6EL8=";
+  };
+  sourceRoot = "source/src";
+
+  patches = [
+    ./toolbox_paths.patch
+  ];
+
+  vendorSha256 = "06s97kpbw40571jjp96jpld1qxb2frd4akcrwwxi1minvs24lb5p";
+
+  nativeBuildInputs = [
+    go-md2man
+  ];
+
+  buildInputs = [
+    podman
+  ];
+
+  preConfigure = ''
+    substituteInPlace cmd/create.go \
+      --replace '"/etc/profile.d/toolbox.sh"}' '"${placeholder "out"}/share/profile.d/toolbox.sh"}'
+  '';
+
+  postBuild = ''
+    cd ../
+    mkdir -p $out/man/man1
+    find ./doc -name '*.md' | while read markdown_path; do
+      go-md2man \
+        -in $markdown_path \
+        -out $out/man/man1/$(basename -s .md $markdown_path)
+    done
+  '';
```

Comment:
Could merge post{build,install} and also use `installShellFiles`.
 ```nix
  postInstall = ''
    cd $sourceRoot
    for d in doc/*.md; do
      go-md2man -in $d -out ''${d%.md}
    done
    installManPage doc/*.[1-9]
    installShellCompletion --bash completion/bash/toolbox
    install profile.d/toolbox.sh -Dt $out/share/profile.d
  '';
```

Reply:
Since I got rid of source root, is `cd ..` ok?

Reply:
Can still use `$sourceRoot`, it defaults to the build dir.

Reply:
Hmm, when I do that, I get the following error

```
❯ nix build ".#toolbox"
warning: Git tree '/home/mjlbach/Repositories/nix/nixpkgs' is dirty
error: --- Error ---------------------------------------------------------------------------------------------------------------------------------------------------------------- nix
builder for '/nix/store/c2jkc3gs4by287nj42pkl5jgqxhd57w1-toolbox-0.0.99.drv' failed with exit code 1; last 10 log lines:
  github.com/containers/toolbox/cmd
  github.com/containers/toolbox
  Building subPackage ./cmd
  Building subPackage ./pkg/podman
  Building subPackage ./pkg/shell
  Building subPackage ./pkg/utils
  Building subPackage ./pkg/version
  running tests
  installing
  /nix/store/l5gksj4fh6ipwk58aiavn9c69p05lh9y-stdenv-linux/setup: line 81: cd: source: No such file or directory
```

Reply:
Default sourceRoot is only the dirname and not the path, sorry.

Reply:
Oh no worries, just wanted to make sure I wasn't messing something up 😆